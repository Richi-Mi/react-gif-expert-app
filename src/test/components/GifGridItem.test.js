import '@testing-library/jest-dom'
import React from 'react'
import GifGridItem from '../../components/GifGridItem'

import { shallow } from 'enzyme'

// Enzyme-to-json nos va a servir para hacer las snapshots de componentes de React
describe(' 1.- Pruebas en <GifGridItem />' , () => {
    test(' - Debe de Mostrar <GifGridItem /> correctamente', () => {
        const wrapper = shallow( <GifGridItem /> ) // Renderizamos el componente para usarlo como html
        
        expect( wrapper ).toMatchSnapshot();
    })
})
// El archivo de setUpTests.js dejarlo, ese es el que va a tomar node, para configuras los test