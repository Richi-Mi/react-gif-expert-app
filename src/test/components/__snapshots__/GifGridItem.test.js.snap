// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1.- Pruebas en <GifGridItem />  - Debe de Mostrar <GifGridItem /> correctamente 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img />
  <p>
     
    . 
  </p>
</div>
`;
